code/modules/projectiles/projectile/beams.dm

```diff
@@ -55,28 +55,60 @@ In pvp they also have more lasting damages, such as infections, pain form burns,
 	armor_penetration = 15 //Some AP
 	recoil = 2
 
+// Laser bullets are your premium and expensive upgrade to your traditional bullets.
+// The basic design philosophy here is that they possess the AP of HV and the damge of FMJ all while having half the recoil.
+// Keep in mind that laser bullets do not imbed and leave no shrapnel behind, which is stupidly lethal as of date.
+
+
+// 223 is a middle ground between 408 and 75. Damage of 408, AP of 75
+/obj/item/projectile/beam/laser_223
+	damage_types = list(BRUTE = 20)
+	armor_penetration = 20
+	recoil = 5
+	wounding_mult = WOUNDING_SERIOUS
+
+/obj/item/projectile/beam/laser_223/ap
+	damage_types = list(BRUTE = 16)
+	armor_penetration = 36
+	recoil = 7
+	wounding_mult = WOUNDING_NORMAL
+
+/obj/item/projectile/beam/laser_223/lethal
```

Comment:
Fundementally this PR was aimed to revert the removal for everything. The quastion as to why HP is an interresting one. I really don't have access to the original thought and idea way back then so I may only guess as to why laser HP was added but simply plain argued is that it simply adds another layer of depth and options players may use to prepare against specific scenarios. Being able to shred giant roach hordes with a single shot does sound super satisfying.

Reply:
Adding the laser bullets back is fine, it was a dumb removal.

Adding laser bullets back and aping regular bullets is....uninspired. Do something else

Reply:
I am going to give AP an xray version, penetrating walls. (I have to make a second PR to fix Xray not penetrating walls).
And HP a laser version that deflects if armor wins against it in a roll.
This will make it unique to both aspects without being super OP on both ends. 

I took radiation as an option but because of how lethal it is in PVP only, I chose to skip on it. Dealing with radiation often just leads to fustration from what I have observed, especially so if medical is not around.

Reply:
> I am going to give AP an xray version, penetrating walls. (I have to make a second PR to fix Xray not penetrating walls). And HP a laser version that deflects if armor wins against it in a roll. This will make it unique to both aspects without being super OP on both ends.
> 
> I took radiation as an option but because of how lethal it is in PVP only, I chose to skip on it. Dealing with radiation often just leads to fustration from what I have observed, especially so if medical is not around.

This will take me a little bit to get done and working.